pf/internal/schemashim/custom_type_test.go

```diff
@@ -109,10 +109,10 @@ func TestCustomListAttribute(t *testing.T) {
 	shimmed := &attrSchema{"key", pfutils.FromAttrLike(raw)}
 	assert.Equal(t, shim.TypeList, shimmed.Type())
 	assert.NotNil(t, shimmed.Elem())
-	_, isPseudoResource := shimmed.Elem().(shim.Schema)
+	_, isPseudoResource := shimmed.Elem().(shim.Resource)
```

Comment:
Assert on next line interestingly said it expects Resource, but the cast is casting to shim.Schema.